### PR TITLE
Issue970 update user provisioned metric preamble 2

### PR DESCRIPTION
**Pull request by xcoulon**
- refactor metrics
- remove commented out code
- refactor toolchain status controller tests

### CODE REVIEW

File: pkg/controller/toolchainstatus/toolchainstatus_controller_test.go [hunk 1]
```diff
@@ -453,7 +450,8 @@ func TestToolchainStatusConditions(t *testing.T) {
 			defer counter.Reset()
 			toolchainStatus := NewToolchainStatus()
 			toolchainStatus.Status.Members = []toolchainv1alpha1.Member{
-				memberClusterSingleReady(),
+				memberCluster("member-1", ready(), userAccountCount(10)),
+				// memberCluster("member-2", ready()),
```
Comment (alexeykazakov) [inline pkg/controller/toolchainstatus/toolchainstatus_controller_test.go:454]: ?

**Review by alexeykazakov — COMMENTED** [review]
I haven't read it very carefully but LGTM overall.

**Review by alexeykazakov — APPROVED** [review]

File: pkg/controller/toolchainstatus/toolchainstatus_controller_test.go (hunk 1 above)
Comment (xcoulon) [reply pkg/controller/toolchainstatus/toolchainstatus_controller_test.go:454]: right, that's a left-over 😬

Comment (xcoulon) [reply pkg/controller/toolchainstatus/toolchainstatus_controller_test.go]: removed in 953ad36

**Pull request merged**